lib/perl/Genome/Model/Build.pm

```diff
@@ -1359,8 +1361,8 @@ sub set_build_id {
 }
 
 sub _launch {
-    my $self = shift;
-    my %params = @_;
+    my ($self, $params, $workflow_xml) = @_;
+    my %params = %$params;
```

Comment:
`%params` is mostly passed by reference below.  Those could be changed to use `$params` instead and then maybe this variable isn't necessary.  (Should the `$workflow_xml` be added as a param instead of passed separately?  Or validated some other way?)